<launch>

  <arg name="ground_truth_vertex_labels_file" 
    default="/media/scratch1/albanesg/scans/scene0011_01/panoptic_vertex_labels.txt"/>
  <arg name="ground_truth_pointcloud_file" 
    default="/media/scratch1/albanesg/scans/scene0011_01/scene0011_01_vh_clean_2.ply"/>

  <node name="evaluation_manager_node" pkg="panoptic_mapping_evaluation" type="evaluation_manager_node.py" required="true">
    <param name="data_dir" value="/media/scratch1/albanesg/runs/scene0011_01/test"/>
    <param name="export_eval_data_srv_name" value="export_evaluation_data"/>
    <param name="ground_truth_vertex_labels_file" value="$(arg ground_truth_vertex_labels_file)"/>
  </node>

  <node name="evaluation_data_export" pkg="panoptic_mapping_evaluation" type="evaluation_data_export_node">
        <param name="ground_truth_pointcloud_file" value="$(arg ground_truth_pointcloud_file)"/>
        <param name="is_single_tsdf" value="false"/>
  </node>
</launch>